integration-tests/src/aave_router.rs

```diff
@@ -95,7 +95,7 @@ fn with_stablepool(execution: impl FnOnce(AssetId)) {
 		assert_ok!(Stableswap::create_pool(
 			hydradx_runtime::RuntimeOrigin::root(),
 			pool,
-			[DOT, ADOT].to_vec(),
+			BoundedVec::truncate_from([DOT, ADOT].to_vec()),
```

Comment:
Instead of using truncate_from(), I would use `try_into().unwrap()` for most tests.

When the list has only 1-2 items, it's fine, but when it has more, it would be better for the tests to fail when the bound limit changes, rather than silently truncating it.

Reply:
no, i dont agree here. these are existing tests, and you can clearly see that it only uses two assets. no need to use try into. 
in fact, you never need to use it.  you were not be able to create pool with more assets before ,   the create pool would return error. And in integration tests you dont need to test it if you can create pool with more assets, you usually only setting up correct pools.